frontend/public/components/alert.tsx

```diff
@@ -229,19 +234,15 @@ const ActiveAlerts = ({alerts}) => <div className="co-m-table-grid co-m-table-gr
     <div className="col-xs-2">Value</div>
   </div>
   <div className="co-m-table-grid__body">
-    {alerts.map((a, i) => {
-      const name = a.labels.alertname;
-      const description = _.get(a, 'annotations.description') || _.get(a, 'annotations.message') || name;
-      return <ResourceRow key={i} obj={a}>
-        <div className="col-xs-6 co-resource-link-wrapper">
-          <Cog options={[silenceAction(a)]} />
-          <Link className="co-resource-link" to={detailsURL(AlertResource, name, a.labels)}>{description}</Link>
-        </div>
-        <div className="col-xs-2"><Timestamp timestamp={a.activeAt} /></div>
-        <div className="col-xs-2"><State state={a.state} /></div>
-        <div className="col-xs-2">{a.value}</div>
-      </ResourceRow>;
-    })}
+    {_.sortBy(alerts, alertDescription).map((a, i) => <ResourceRow key={i} obj={a}>
```

Comment:
Maybe just

```suggestion
    {_.sortBy(alerts, ['annotations.description', 'annotations.message', 'labels.alertname']).map((a, i) => <ResourceRow key={i} obj={a}>
```

Reply:
I'm actually using `alertDescription()` a couple of lines below too, so I think it's a little better to have it as a separate function.